dist/angular-ui-tree.js

```diff
@@ -441,6 +442,13 @@
               }
             });
 
+            scope.$watch(attrs.dragElementExtraWidth, function (val) {
```

Comment:
Could you add a comment that identifies when this value could change, I assume that it will be passed in from the calling code as an angular variable.